source/manual/alerts/rabbitmq-high-number-of-unprocessed-messages.html.md

```diff
@@ -0,0 +1,45 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'RabbitMQ: High number of unprocessed messages'
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2020-09-07
+review_in: 6 months
+---
+
+For information about how we use RabbitMQ, see [here][rabbitmq_doc].
+
+We check that there is not a significant build up of messages on certain queues
+as we expect messages to be processed quickly. The queues this alert applies to
+are: `email_alert_service`, `email_unpublishing` and `cache_clearing_service-*`
+(low, medium and high). The plugin which implements this alert is
+[here][plugin].
+
+The Icinga check is performed by connecting to RabbitMQ's admin API and
+triggering if the number of messages on the queues listed above crosses certain
+thresholds: 25 for critical and 5 for warning. These thresholds are set in
+[puppet][thresholds].
```

Comment:
Minor: I usually try to link to the code at a specific commit, so that the link still works if the content changes.

Reply:
Sure 👍